### PR TITLE
Update test command

### PR DESCRIPTION
**Pull request by gbrener**
Update one of the test commands to work in the latest version of conda / conda-build. This line was causing a failure because the _test environment no longer exists.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ requirements:
 test:
     commands:
         - iconv --help  # [not win]
-        - conda inspect linkages -n _test libiconv  # [linux]
+        - conda inspect linkages -p $CONDA_PREFIX libiconv  # [linux]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: I believe that `$PREFIX` is enough.
Can you also change `# [linux]` to `# [not win]` so this runs on `OS X` too?

**Comment by jakirkham** [comment]
Needs to be [re-rendered]( https://github.com/conda-forge/conda-smithy/tree/777aac37e95a9dd9d43f839d08f377968de28be9#re-rendering-an-existing-feedstock ).

**Comment by ocefpaf** [comment]
@gbrener we will be able to merge this only after conda-forge moves to `conda-build 2`.

```shell
The following packages will be DOWNGRADED due to dependency conflicts:

    conda-build: 2.0.1-py35_0 defaults --> 1.21.14-py35_0 defaults
```

**Comment by ocefpaf** [comment]
Done in #12  Thanks @gbrener

**Pull request closed without merging**